rules/linux/lnx_network_service_scanning.yml

```diff
@@ -0,0 +1,42 @@
+action: global
+title: Linux Network Service Scanning
+id: 3e102cd9-a70d-4a7a-9508-403963092f31
+status: experimental
+description: Detects enumeration of local or remote network services.
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/21
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1046/T1046.md
+falsepositives:
+  - Legitimate administration activities
+level: medium
+tags:
+  - attack.discovery
+  - attack.t1046
+---
+logsource:
+  category: process_creation
+  product: linux
+detection:
+  selection:
+    ProcessName|endswith:
+      - '/nmap'
+      - '/telnet'
+      - '/nc'
+      - '/netcat'
+  condition: selection
+---
+logsource:
+  product: linux
+  service: auditd
+  definition: 'Configure these rules https://github.com/Neo23x0/auditd/blob/master/audit.rules#L182-L183'
+detection:
+  selection:
+    type: 'SYSCALL'
+    exe|endswith:
+      - '/nc'
+      - '/netcat'
+      - '/telnet'
+      - '/nmap'
```

Comment:
I believe that we could leave here only netcat stuff, as the rest of things will be converted to auditd from `process_execution`, and telnet/nmap will generate network connection anyway.
As you might remember, we went this way to focus only on network connections creation by netcat, to filter out local port binding

Reply:
sorry, I probably mislead you. I don't remember why I was thinking thins way honestly :D 
I fixed the rule to include all options. thank you!